packages/web-app-files/src/components/BatchActions.vue

```diff
@@ -227,11 +259,24 @@ export default {
 
     isEmpty() {
       return this.activeFilesCurrentPage.length < 1
+    },
+
+    isDangerVariation() {
+      return this.inContextActions ? 'passive' : 'danger'
+    },
+
+    isOutlineAppearance() {
+      return this.inContextActions ? 'raw' : 'outline'
+    },
+
+    isPrimaryVariation() {
+      return this.inContextActions ? 'passive' : 'primary'
     }
   },
 
   methods: {
     ...mapActions('Files', ['removeFilesFromTrashbin', 'resetFileSelection']),
+    ...mapActions('Files/sidebar', { openSidebar: 'open' }),
```

Comment:
If you decide to use the mixin (as stated in another comment), this line would not be needed anymore - solved by the trigger method in the mixin.

Reply:
Pretty sure this has been resolved by you by now, marking as resolved